src/account/account.entity.ts

```diff
@@ -49,9 +53,19 @@ export class AccountEntity implements Account {
         public readonly apId: URL,
         public readonly apFollowers: URL | null,
         public readonly isInternal: boolean,
+        private events: AccountEvent[],
     ) {}
 
-    static create(data: Data<Account>) {
+    static pullEvents(account: Account): AccountEvent[] {
+        if (account instanceof AccountEntity) {
```

Comment:
We need this `instanceof` because we pass `Account` not `AccountEntity` - and we need to pass that because that is what the repository receives